assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -14,8 +14,311 @@
+      MAIN_TEXT: __('Too much text on page', 'web-stories'),
+      // eslint-disable eslint/no-useless-escape
+      HELPER_TEXT: sprintf(
+        /* translators: 1: maximum number of story characters. 2: maximum percentage of characters, depending on number of pages. */
+        __(
+          'Keep text to max %1$d characters per page. Consider using a page attachment, breaking up the text into multiple screens, or keeping the total number of pages with a lot of text to less than %2$d%% percent of the pages in the story.',
+          'web-stories'
+        ),
+        MAX_STORY_CHARACTERS,
+        MAX_CHARACTER_PERCENTAGE
+      ),
+    },
+    TOO_LITTLE_TEXT: {
+      MAIN_TEXT: __('Too little text', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum number of story characters */
+        __(
+          'The entire story should have a minimum of %1$d characters of text in total.',
+          'web-stories'
+        ),
+        MIN_STORY_CHARACTERS
+      ),
+    },
+  },
+  MEDIA: {
+    VIDEO_IMAGE_TOO_SMALL_ON_PAGE: {
+      MAIN_TEXT: __('Video or image too small on the page', 'web-stories'),
+      HELPER_TEXT: __(
+        // todo add [more info](https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/#visual-treat)
+        'Use full screen videos and images where possible to keep to an immersive feeling.',
+        'web-stories'
+      ),
+    },
+    LOW_IMAGE_RESOLUTION: {
+      MAIN_TEXT: __('Low image resolution', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum image size width 2: minimum image size height */
+        __(
+          'Use %1$d x %2$dpx for full screen images, keep to similar pixel density for cropped images.',
+          'web-stories'
+        ),
+        IMAGE_SIZE_WIDTH,
+        IMAGE_SIZE_HEIGHT
+      ),
+    },
+    VIDEO_TOO_LONG: {
+      MAIN_TEXT: __('Video too long', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: maximum video length in minutes */
+        _n(
+          'Break longer videos into segments of %1$d minute or less.',
+          'Break longer videos into segments of %1$d minutes or less.',
+          MAX_VIDEO_LENGTH_MINUTES,
+          'web-stories'
+        ),
+        MAX_VIDEO_LENGTH_MINUTES
+      ),
+    },
+    VIDEO_FRAME_RATE_TOO_LOW: {
+      MAIN_TEXT: __('Video frame rate low', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum number of frames per second for video */
+        __(
+          'Video should have a minimum of %1$d frames per second.',
+          'web-stories'
+        ),
+        MIN_VIDEO_FPS
+      ),
+    },
+    VIDEO_RESOLUTION_TOO_LOW: {
+      MAIN_TEXT: __('Video resolution low', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum video resolution */
+        __('Videos should have a minimum resolution of %1$dp.', 'web-stories'),
+        MIN_VIDEO_RESOLUTION
```

Comment:
No need for nummbered placeholders if there is only one.

Also, move the `p` to the variable to avoid accidental translation or omission by translators. 

```suggestion
        /* translators: %s: minimum video resolution. */
        __('Videos should have a minimum resolution of %s.', 'web-stories'),
        `${MIN_VIDEO_RESOLUTION}p`
```